src/sas/qtgui/Perspectives/Fitting/SmearingWidget.py

```diff
@@ -355,7 +363,9 @@ def onPinholeSmear(self):
             backend_accuracy = ACCURACY_DICT.get(accuracy)
             if backend_accuracy:
                 self.current_smearer.set_accuracy(accuracy=backend_accuracy)
-
+            else:
+                self.current_smearer.set_accuracy(accuracy='low')
+    
```

Comment:
Why is this necessary? ACCURACY_DICT contains all possible keys, and in case we don't find the right key, I think it is better to just stay with the previously defined accuracy level, rather than reset it quietly?

Reply:
I just reused the same code already present in def onPinholeSmear. But note that when a data set is loaded or a model selected, getSmearInfo is called without the smearing interface being "activated", so at this point backend_accuracy is None and the if ... else is needed, I think.